login.html

```diff
@@ -0,0 +1,97 @@
+        <div class="container-fluid">
+          <a class="navbar-brand" href="index.html">BeSocial</a>
+          <button
+            class="navbar-toggler"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarSupportedContent"
+            aria-controls="navbarSupportedContent"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="index.html">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="profile.html">Profile</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="login.html">LogIn</a>
+              </li>
+            </ul>
+            <form class="d-flex" role="search">
+              <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search" />
+              <button class="btn btn-outline-success" type="submit">Search</button>
+            </form>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <div class="d-flex justify-content-center mb-3">
+        <h1>LogIn</h1>
+      </div>
+      <div class="d-flex justify-content-center">
+        <div class="tab-content">
+          <div class="tab-pane fade show active" id="pills-login" role="tabpanel" aria-labelledby="tab-login">
+            <form class="d-flex flex-column">
+              <!-- Email input -->
+              <div class="form-outline mb-4">
+                <input type="email" id="loginName" class="form-control" required />
+                <label class="form-label" for="loginName">Email or username</label>
+              </div>
+              <div class="form-outline mb-4">
+                <input type="password" id="loginPassword" class="form-control" required />
+                <label class="form-label" for="loginPassword">Password</label>
+              </div>
+              <div class="mb-4 d-flex justify-content-between">
+                <div class="d-flex justify-content-center pe-3">
+                  <div class="form-check mb-3 mb-md-0">
+                    <input class="form-check-input" type="checkbox" value="" id="loginCheck" checked />
+                    <label class="form-check-label" for="loginCheck"> Remember me </label>
+                  </div>
+                </div>
+
+                <div class="d-flex justify-content-center">
+                  <a href="#!">Forgot password?</a>
+                </div>
+              </div>
+              <button type="submit" formaction="/profile.html" class="btn btn-primary btn-block mb-4 mx-auto">
```

Comment:
```suggestion
              <button type="submit"  class="btn btn-primary btn-block mb-4 mx-auto">
```

Form action should go on the `<form>` tag. This works and it's not wrong, but it's not what was requested.

Reply:
Implemented this change.